integration/auth/user_mgt_test.go

```diff
@@ -411,3 +414,27 @@ func testDeleteUsers(t *testing.T) {
 		}
 	}
 }
+
+func TestGetUserInvalidId(t *testing.T) {
```

Comment:
Suggestion: This sort of logic is easier/cheaper to verify in unit tests. Also these tests would fail if somebody creates a user with this email/phone number in the project. Consider just updating https://github.com/firebase/firebase-admin-go/blob/master/auth/user_mgt_test.go#L143 and removing these.